server/routerlicious/packages/routerlicious/src/scriptorium/index.ts

```diff
@@ -79,23 +48,5 @@ export async function create(config: Provider): Promise<IPartitionLambdaFactory>
 			: opCollection.createTTLIndex({ mongoTimestamp: 1 }, mongoExpireAfterSeconds));
 	}
 
-	executeOnInterval(
-		async () =>
```

Comment:
I understand we don't exercise this feature, but other R11s consumers may need it?

Reply:
What is other R11s consumer you are refering @yangg-msft ? Real customer or code consumer? If code consumer, then there is no other consumer, if it is a concern for real customer, then I can add it back

Reply:
Readded this piece of code. Changed the execution more reasonable way but it still won't work